website-custom-views/src/content/development/permissions.mdx

```diff
@@ -99,22 +83,22 @@ const CustomViewRoutes = () => {
 
 ## In components
 
-Similarly, you can also evaluate user permissions in your React components, for example to deactivate a button.
+You can evaluate user permissions in your React components, for example to deactivate a button.
 
 ```jsx
 import { useIsAuthorized } from '@commercetools-frontend/permissions';
 import PrimaryButton from '@commercetools-uikit/primary-button';
-import { PERMISSIONS } from '../constants';
+import { PERMISSIONS_WITH_GROUPS } from '../constants';
```

Comment:
@Anshuman71 Just curious, how did you know to change this?

Reply:
I changed this just to use the permissions with groups with the `useIsAuthorized` hook because we didn't have any examples of this. I was looking for such an example, so I thought it would make sense to add here.